langchain/spark_sql.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
```

Comment:
nit: lets move this to utilities

Reply:
(as in the whole file should go to langchain/utilities/spark_sql, not that specific line)

Reply:
yeah that whole file

Reply:
Make sense. This is done.